include/chibi/sexp.h

```diff
@@ -1445,7 +1445,7 @@ enum sexp_uniform_vector_type {
 #define sexp_context_max_depth(x) (sexp_vector_ref(sexp_context_specific(x), SEXP_FIVE))
 #define sexp_context_exception(x) (sexp_vector_ref(sexp_context_specific(x), SEXP_SIX))
 
-#if SEXP_USE_ALIGNED_BYTECODE
+#ifdef SEXP_USE_ALIGNED_BYTECODE
```

Comment:
This is only required because you removed the line above where it's set to 0.

Reply:
Actually no, that line was erroring compilation even when all I'd done was uncomment the `#define`. I don't know if I have a more strict compiler than you do but for me it was either this or `#if SEXP_USE_ALIGNED_BYTECODE == 1`.